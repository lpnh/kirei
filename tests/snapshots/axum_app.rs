#[test]
fn layout() {
    format_and_snapshot!("axum-app/_layout.html");
}

#[test]
fn error() {
    format_and_snapshot!("axum-app/error.html");
}

#[test]
fn greet() {
    format_and_snapshot!("axum-app/greet.html");
}

#[test]
fn index() {
    format_and_snapshot!("axum-app/index.html");
}
