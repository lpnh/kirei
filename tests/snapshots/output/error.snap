---
source: tests/snapshots/axum_app.rs
description: "{% extends \"_layout.html\" %}\n{%- block title -%}\n{%- match err -%}\n{% when AppError::NotFound -%} 404: Not Found\n{% else -%} 500: Internal Server Error\n{%- endmatch -%}\n{%- endblock -%}\n{%- block content -%}\n<h1>\n{%- match err -%}\n{% when AppError::NotFound -%} 404: Not Found\n{% else -%} 500: Internal Server Error\n{%- endmatch -%}\n</h1>\n{%- match err -%}\n{% when AppError::NotFound -%}\n{% when AppError::Render(err) -%}\n<pre>{{ err }}</pre>\n{%- endmatch -%}\n<h2><a href=\"/\">Back to the first page.</a></h2>\n{%- endblock -%}\n"
---
{% extends "_layout.html" %}
{%- block title -%}
    {%- match err -%}
        {% when AppError::NotFound -%} 404: Not Found
        {% else -%} 500: Internal Server Error
    {%- endmatch -%}
{%- endblock -%}
{%- block content -%}
    <h1>
        {%- match err -%}
            {% when AppError::NotFound -%} 404: Not Found
            {% else -%} 500: Internal Server Error
        {%- endmatch -%}
    </h1>
    {%- match err -%}
        {% when AppError::NotFound -%}
        {% when AppError::Render(err) -%} <pre>{{ err }}</pre>
    {%- endmatch -%}
    <h2><a href="/">Back to the first page.</a></h2>
{%- endblock -%}
